### ISSUE TITLE
Zoom Videos Posted On Facebook

### ISSUE THREAD

**Issue by wwongcand** [igrigorik/videospeed#860]
Some Zoom videos posted on Facebook feeeds do not have the option to use the video speed controller. Is there a fix for this? Thanks!

**Comment by pepijnmm** [comment]
Do you have a video as an example?
the code is open-source so anyone can make a fix for it in theory. But not sure if it will happen it depends.

**Comment by wwongcand** [comment]
Yes. This video is an example - https://www.facebook.com/brian.grasso/videos/343808050826387/. It's in a group so not sure if you'll be able to view it or look into it from just the link.

**Comment by thewheat** [comment]
@wwongcand seems that the link is no longer valid or that the video is set to share to only specific people 😅  Do you have any other example?

![image](https://user-images.githubusercontent.com/892961/133922231-1c818fce-a22f-473e-9b0e-8c776fac967a.png)
